packages/framework/block-std/src/gfx/controller.ts

```diff
@@ -20,21 +23,27 @@ import {
 } from './surface/element-model.js';
 import { Viewport } from './viewport.js';
 
-export class GfxController {
+export class GfxController extends LifeCycleWatcher {
   private _disposables: DisposableGroup = new DisposableGroup();
 
   private _surface: SurfaceBlockModel | null = null;
 
+  static override key = 'gfxController';
+
   readonly grid: GridManager;
```

Comment:
Will the grid manager and layer manager also be extensions in the future?

Reply:
I don't think so. These two are required modules that are part of the `GfxController`.